.github/workflows/format.yml

```diff
@@ -1,35 +0,0 @@
-name: Check Make Format
```

Comment:
Is it no problem to remove this checking?
This ci action is added in #230.

Reply:
As I know, the purpose of #230 is to check the imports order. 
Does the new linter and lint rules also check this imports order?
And the lint rules should avoid checking `statik.go` imports order.